subprojects/gst-plugins-bad/sys/d3d11/gstd3d11compositor.cpp

```diff
@@ -801,6 +822,41 @@ gst_d3d11_compositor_pad_set_property (GObject * object, guint prop_id,
           (GstD3D11CompositorSizingPolicy) g_value_get_enum (value);
       pad->position_updated = TRUE;
       break;
+    case PROP_PAD_CROP:
+    {
+        GST_DEBUG_OBJECT(pad, "setting crop parameters");
+       if (pad->crop_properties != NULL)
+        {
+            gst_structure_free(pad->crop_properties);
+            pad->crop_properties = NULL;
+        }            
+        pad->crop_properties =
```

Comment:
this structure is still leaked if the property is set, but element didn't switch to play (in other words, if pad->convert == NULL)

Reply:
if you store it, you need to free it in the "dispose" of the class

Reply:
Ohh, Okay I missed that out. 
I will check with finalize method in it.